aws/resource_aws_cloudwatch_log_metric_filter.go

```diff
@@ -139,9 +139,7 @@ func resourceAwsCloudWatchLogMetricFilterRead(d *schema.ResourceData, meta inter
 
 	d.Set("name", mf.FilterName)
 	d.Set("pattern", mf.FilterPattern)
-	d.Set("metric_transformation", flattenCloudWatchLogMetricTransformations(mf.MetricTransformations))
-
-	return nil
+	return d.Set("metric_transformation", flattenCloudWatchLogMetricTransformations(mf.MetricTransformations))
```

Comment:
Nit: Our typical convention for error checking these is to return error context and leave `return nil` last 😄 

```suggestion
	if err := d.Set("metric_transformation", flattenCloudWatchLogMetricTransformations(mf.MetricTransformations)); err != nil {
		return fmt.Errorf("error setting metric_transformation: %s", err)
	}

	return nil
```